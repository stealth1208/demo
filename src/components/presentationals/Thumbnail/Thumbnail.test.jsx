import React from 'react';
import renderer from 'react-test-renderer';
import { Thumbnail } from '../index';

it('renders correctly', () => {
    const tree = renderer
      .create(<Thumbnail src="src.jgp" alt="" onClick={() => console.log('onClick')}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });