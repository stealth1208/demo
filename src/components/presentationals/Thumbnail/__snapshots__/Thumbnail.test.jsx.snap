// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className=" character-thumbnail"
  onClick={[Function]}
>
  <img
    alt=""
    src="."
    width="100%"
  />
</div>
`;
